test/powershell/engine/Jobs.Tests.ps1

```diff
@@ -0,0 +1,17 @@
+Describe 'Basic Job Tests' -Tags 'CI' {
+
+    $job = Start-Job {1}
```

Comment:
This line should be in a `BeforeAll` block.

Also, I think the job should be removed in a `AfterAll` block.